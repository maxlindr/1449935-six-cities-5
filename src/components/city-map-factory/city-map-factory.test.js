import React from 'react';
import renderer from 'react-test-renderer';
import cityMapFactory, {CityMapType} from './city-map-factory';

test(`cityMapFactory должена возвращать корректно отображаемый компонент`, () => {
  const CityMap = cityMapFactory(CityMapType.MAIN);

  const component = renderer.create(
      <CityMap />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
