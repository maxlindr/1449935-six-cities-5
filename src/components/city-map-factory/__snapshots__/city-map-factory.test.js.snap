// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cityMapFactory должена возвращать корректно отображаемый компонент 1`] = `
<section
  className="cities__map map"
/>
`;
